odoo/addons/base/models/res_partner.py

```diff
@@ -239,6 +239,42 @@ def default_get(self, default_fields):
         ('check_name', "CHECK( (type='contact' AND name IS NOT NULL) or (type!='contact') )", 'Contacts require a name'),
     ]
 
+    @api.depends('name', 'user_ids', 'image_1920')
+    def _compute_avatar_1920(self):
+        self._compute_avatar('avatar_1920', 'image_1920')
```

Comment:
Seems duplicate code ? You may change depends and call super() instead of redefining the code itself (as there is quite some noise maybe there are some differences I did not spot, but code seems duplicated for all compute methods.).

Reply:
Agree here, better call super than repeat its code. Too bad we don't have a better syntax to just update dependencies.